packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -398,7 +397,8 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     if (resultCode != Activity.RESULT_OK) {
-      finishWithError(ERROR_REASON_CANCELED, String.valueOf(resultCode));
+      // User cancelled so we return a null auth object.
```

Comment:
Should this only happen if `resultCode == Activity.RESULT_CANCELED`? (Not sure what other failure reasons could be...)